src/slices/Glossary/Glossary.tsx

```diff
@@ -90,7 +91,9 @@ export const Glossary: React.FC<GlossaryProps> = ({ slice }: GlossaryProps) => {
                         onClick={async () => handleAnchorClick(item.slug)}
                       />
                     </Flex>
-                    <Text color="black">{item.text}</Text>
+                    <Text color="black" style={{ wordBreak: 'break-word' }}>
```

Comment:
This should also be possible:
```suggestion
                    <Text color="black"  wordBreak="break-word">
```